src/include/execution/sql/functions/string_functions.h

```diff
@@ -29,8 +29,9 @@ class StringFunctions {
     Length(result, ctx, str);
   }
 
-  /** Compute CONCAT(left, right). */
-  static void Concat(StringVal *result, exec::ExecutionContext *ctx, const StringVal &left, const StringVal &right);
+  /** Compute CONCAT(str1, str2, str3, ...). */
+  static void Concat(StringVal *result, exec::ExecutionContext *ctx, const int64_t num_inputs,
```

Comment:
Should make the input count unsigned, probably a `uint32_t`.

Reply:
Fixed